src/index.css

```diff
@@ -16,3 +16,8 @@ body {
     -moz-osx-font-smoothing: grayscale;
 }
 
+.myLineDecoration {
```

Comment:
I don't think this style is used anywhere, is it?

Reply:
removed